tests/phpunit/tests/test-languages-list.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+class Languages_List_Test extends PLL_UnitTestCase {
+	/**
+	 * @param WP_UnitTest_Factory $factory
+	 */
+	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
+		parent::wpSetUpBeforeClass( $factory );
+
+		self::create_language( 'en_US' );
```

Comment:
Maybe you should do this in `set_up()` as you'll corrupt in a test.

Reply:
Indeed! Done in f0efa68c.